src/ensembl/stories/shared-components/radio/Radio.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import Radio, { RadioOptions } from 'src/shared/components/radio/Radio';
+import { storiesOf } from '@storybook/react';
+import styles from './Radio.stories.scss';
+import { action } from '@storybook/addon-actions';
+
+const radioData: RadioOptions = [
+  { value: 'default', label: 'Default' },
+  { value: 'length_longest', label: 'Spliced length: longest - shortest' },
+  { value: 'length_shortest', label: 'Spliced length: shortest - longest' }
+];
+
+const Wrapper = (props: any) => {
+  const [selectedRadio, setselectedRadio] = useState('default');
+
+  const handleOnchange = (value: string) => {
+    setselectedRadio(value);
+    action('radio-clicked')(value);
+  };
+
+  return (
+    <div>
+      <Radio
+        {...props}
+        options={radioData}
+        onChange={handleOnchange}
+        selectedOption={selectedRadio}
+      ></Radio>
+    </div>
+  );
+};
+
+storiesOf('Components|Shared Components/Radio', module)
+  .add('default', () => {
+    return <Wrapper />;
+  })
+  .add('Entity Viewer style', () => {

```

Comment:
I am not sure there is much point of having a dedicated story for EntityViewer background — it doesn't show us anything new except for how the radio element can accept custom classes from the parent. For example, we don't have such stories for our checkboxes.

Reply:
Yep i wasn't sure about this one. So just the default one should be enough, is that right?

Reply:
> So just the default one should be enough, is that right?

Yes, I think so.